stdlib/_typeshed/__init__.pyi

```diff
@@ -348,6 +349,8 @@ TraceFunction: TypeAlias = Callable[[FrameType, str, Any], TraceFunction | None]
 #   https://github.com/microsoft/pyright/issues/4339
 class DataclassInstance(Protocol):
     __dataclass_fields__: ClassVar[dict[str, Field[Any]]]
+    if sys.version_info >= (3, 13):
```

Comment:
I don't think this needs to be part of the protocol.

Reply:
commented above, it could be useful for narrowing with `is_dataclass`, but otherwise can remove it.

Reply:
https://github.com/python/typeshed/pull/12265/commits/4dabac79962038e3e5b83a7cd54edb7bb6b4cb38